src/components/UNSTABLE__LayoutModules/ICAModule/index.scss

```diff
@@ -0,0 +1,42 @@
+////
+/// ICA module.
+/// @group ica-module
+/// @copyright IBM Security 2020
+////
+
+@import '@carbon/layout/scss/spacing';
+@import '@carbon/themes/scss/tokens';
+
+@import '../../../globals/namespace/index';
+@import '../../Component/mixins';
+@import '../TitleBarModule/variables';
+@import '../variables';
+
+@include security--component($name: #{$layout-modules__name}--ica) {
+  /// Spacing.
+  /// @type Length
+  $ica-module__spacing: $carbon--layout-01;
+
+  padding-top: $ica-module__spacing;
+  padding-bottom: $ica-module__spacing;
+
+  &--hover {
+    padding-top: $ica-module__spacing;
```

Comment:
this is probably intentional, but just to be sure the heights of your ICAModule will be different depending if it's interactive or not with these.

Reply:
That's exactly right 🎉